.evergreen/config.yml

```diff
@@ -678,11 +678,11 @@ axes:
           DRIVER_REVISION: "main"
           ASTROLABE_EXECUTOR_STARTUP_TIME: 45
       - id: java-master
```

Comment:
Avoided changing the id, because I couldn't prove to myself that it wouldn't have some effect on the waterfall view in Evergreen.